sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/DropwizardRuleHelper.java

```diff
@@ -90,4 +91,9 @@ private DropwizardRuleHelper(Class<A> appClass) {
   public DropwizardAppRule<C> build() {
     return new DropwizardAppRule<>(appClass, configurationBuilder.build());
   }
+
+  @Override
+  public DropwizardAppExtension<C> buildExtension() {
+    return new DropwizardAppExtension<>(appClass, configurationBuilder.build());
+  }
```

Comment:
I don't think that we should add new features to this deprecated class.

Reply:
Since we can't resolve the deprecation of the DwRuleHelper, because we want to keep the JUnit4 support, would it make sense to declare a new class over modifying this existing but deprecated?
If we have a separate class for Jupiter, how can the decision be determined?

Reply:
Nobody should use the DropwizardRuleHelper anymore, it's deprecated. We shouldn't extend a deprecated feature with new stuff?

Reply:
we need to shift this change to another pr, since we dont require it for the environment rule right now